www/artists/dhall/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet">
+    <li class="home">  <a href= "index.html"> Home</a> </li>
```

Comment:
You can't have list items outside of a list of some sort, either ordered or unordered.